transmartApp/build.gradle

```diff
@@ -71,22 +124,22 @@ dependencies {
     testCompile "org.grails:grails-plugin-testing"
     testCompile "org.grails:grails-test"
     testCompile "org.grails.plugins:geb"
-    testCompile "org.grails.plugins:hibernate4:5.0.10"
+    testCompile "org.grails.plugins:hibernate5"
     testCompile 'org.gmock:gmock:0.9.0-r435-hyve2'
     testRuntime "org.seleniumhq.selenium:selenium-htmlunit-driver:2.47.1"
     testRuntime "net.sourceforge.htmlunit:htmlunit:2.18"
     testRuntime 'junit:junit:4.11'
     testRuntime 'org.hamcrest:hamcrest-core:1.3'
     testRuntime 'org.hamcrest:hamcrest-library:1.3'
+
+    //FIXME: this block was in grails.plugins, make sure that the assets keep working
```

Comment:
This should indeed be tested. Previously, it did not work with those dependencies outside of the plugin block.